app/views/image_templates/edit.html.erb

```diff
@@ -8,6 +8,11 @@
   <hr>
 </div>
 
+<% @image_template.match_options['threshold'] ||= 80 %>
```

Comment:
I can compress it down there. The only thing I don't like about it is that feels like it starts to put a bit of logic within what is otherwise very straightforward view code.